macOS/UnitTests/Menus/MoreOptionsMenuTests.swift

```diff
@@ -684,6 +684,92 @@ final class MoreOptionsMenuTests: XCTestCase {
             XCTAssertFalse(printItem.isEnabled, "\(tabContent) expected to not support printing")
         }
     }
+
+    // MARK: - Feedback sub-menu tests
```

Comment:
I’ve added unit tests for the old logic to make sure nothing is impacted after adding the new feature flag.